docs/backends/sqlite3.md

```diff
@@ -53,6 +53,7 @@ The set of parameters used in the connection string for SQLite is:
 * `synchronous` - set the pragma synchronous flag ([link](http://www.sqlite.org/pragma.html#pragma_synchronous))
 * `shared_cache` - should be `true` ([link](http://www.sqlite.org/c3ref/enable_shared_cache.html))
 * `vfs` - set the SQLite VFS used to as OS interface. The VFS should be registered before opening the connection, see [the documenation](https://www.sqlite.org/vfs.html)
+* `foreign_keys` - set the pragma foreign_keys flag ([link](https://www.sqlite.org/pragma.html#pragma_foreign_keys)). Possible values are "on"/"off"
```

Comment:
I'd remove this part:
```suggestion
* `foreign_keys` - set the pragma foreign_keys flag ([link](https://www.sqlite.org/pragma.html#pragma_foreign_keys)).
```
because there are more possible values than this (any of 1, yes, true and on works) and we document `true` for `shared_cache` above and don't document anything at all for `synchronous`.

If we want to improve this, we should document the full set of possible values for all the options and also accept all of them for `shared_cache`, but for now I'd just avoid giving slightly misleading information.